deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -90,6 +90,8 @@ master:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name:
+  
```

Comment:
Probably also a comment here on what this field signifies can be a good addition.